src/tls.c

```diff
@@ -7459,7 +7459,9 @@ static int TLSX_KeyShare_GenEccKey(WOLFSSL *ssl, KeyShareEntry* kse)
         /* Allocate an ECC key to hold private key. */
         kse->key = (byte*)XMALLOC(sizeof(ecc_key), ssl->heap, DYNAMIC_TYPE_ECC);
         if (kse->key == NULL) {
-            WOLFSSL_MSG("EccTempKey Memory error");
+            WOLFSSL_MSG_EX("Failed to allocate %d bytes, ssl->heap: %p",
```

Comment:
Really necessary?

Reply:
In memory constrained environments, I found this very valuable to see how much more memory was needed to avoid a failure.

I did however, leave the original `EccTempKey Memory error` just in case there was some sort of automated monitoring system that looked for that.